addons/binding/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -44,7 +44,7 @@
 					"<dc:title>{2}</dc:title>" +
 					"<upnp:class>{3}</upnp:class>" +
 					"<desc id=\"cdudn\" nameSpace=\"urn:schemas-rinconnetworks-com:metadata-1-0/\">" +
-					"RINCON_AssociatedZPUDN</desc>" +
+					"{4}</desc>" +
```

Comment:
Dan, are you sure this change does not break other stuff? 

From what I saw on the network via WireShark, that field was always set to RINCON_AssociatedZPUDN